spec/dummy/db/schema.rb

```diff
@@ -27,12 +27,11 @@
 
   add_index "queue_classic_jobs", ["q_name", "id"], name: "idx_qc_on_name_only_unlocked", where: "(locked_at IS NULL)", using: :btree
 
-  create_table "queue_classic_later_jobs", force: true do |t|
-    t.string   "q_name"
-    t.string   "method"
+  create_table "queue_classic_later_jobs", id: false, force: :cascade do |t|
```

Comment:
This should not be here anymore, not trace of `queue_classic_later_jobs` should be left I think.